Metro-Records/la-metro-councilmatic
# Upload headshots to S3 via django-storages

## Overview

This PR makes use of an upstream change to `django-councilmatic` allowing the headshot storage backend to be configured by Councilmatic instances. When credentials are provided, it configures the Metro app to store and retrieve headshots from S3.

This PR also performs some housekeeping (#1055):

- Clean up files from old deployment strategies
- Update README to reflect current stack and development practices
- Document environment variables in `app.json`
- Default environment variables to development settings

Connects #1073

### Notes

Depends on new release of django-councilmatic. Current draft pinned to branch with change.

https://github.com/datamade/django-councilmatic/pull/292

## Testing Instructions

- Log in to the review app using the staging credentials in Bitwarden.
- View the board member listing and confirm headshots are rendering. (Most are placeholders. Janice Hahn has a real one.)
- Navigate to a board member and upload a new headshot. Confirm that it appears on their detail page and the listing page.
